src/sql/parts/notebook/outputs/messaging.ts

```diff
@@ -0,0 +1,33 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+export interface MessagePoster {
```

Comment:
You can remove this file entirely. It should not be needed, we don't need inter-process communication with a webview anymore.

Reply:
Ahh. Thanks for pointing this out. Removed this file before I executed git clean command. :)